lib/msf/core/rpc/v10/rpc_module.rb

```diff
@@ -402,6 +402,22 @@ def rpc_target_compatible_evasion_payloads(mname, target)
     res
   end
 
+  # Returns the total modules in each state.
+  #
+  # @return [Hash] Running module stats that contain the following keys:
+  #  * 'ready' [Integer] The number of modules waiting to be kicked off.
+  #  * 'running' [Integer] The number of modules currently in progress.
+  #  * 'results' [Integer] The number of module run/check results.
+  # @exampleHere's how you would use this from the client:
+  #  rpc.call('module.module_stats')
+  def rpc_module_stats
```

Comment:
I'm not a fan of this being called module_stats, it's a bit misleading, it's not stats about a module, and it's a bit redundant with the call being `module.module_stats` 
anyone one else have strong opinions on this?

Reply:
Better alternative could be `state_totals`? Didn't want to call it `running_totals` because it was the only state returned

Reply:
Just to add variety; The original name was just: `running_stats`